src/bindings/python/flux/job/list.py

```diff
@@ -75,6 +79,9 @@ def job_list(
 def job_list_inactive(
     flux_handle, since=0.0, max_entries=1000, attrs=["all"], name=None, queue=None
 ):
+    """
+    List only inactive jobs
+    """
```

Comment:
Following https://peps.python.org/pep-0257/#one-line-docstrings I think it's better to end with a period and to place the opening and closing quotes on a single line.

Reply:
Also unaddressed